src/core/components/operation-tag.jsx

```diff
@@ -54,6 +54,9 @@ export default class OperationTag extends React.Component {
     const DeepLink = getComponent("DeepLink")
     const Link = getComponent("Link")
 
```

Comment:
No need for this empty space here. By convention we want to keep all components access by `getComponents` in one block.